Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -106,5 +110,18 @@ public async Task RemoveResponse(DiscordRequest request, Contexts contexts)
             await _responsesService.RemoveResponse(onEvent, contexts.Server.Id);
             await messageService.SendResponse(x => x.ResponseHasBeenRemoved(contexts, onEvent), contexts);
         }
+
+        [AdminCommand]
+        [DiscordCommand("responses")]
+        public async Task Responses(DiscordRequest request, Contexts contexts)
+        {
+            var possibleArguments = new string[] { "all", "default", "custom" };
```

Comment:
zrób to jako field private readonly

nie ma sensu tworzyć osobnego obiektu dla każdego odpalenia metody